deeptools/plotHeatmap.py

```diff
@@ -186,7 +186,7 @@ def plotMatrix(hm, outFileName,
 
     if zMax is None:
         if matrix_flatten is None:
-            matrix_flatten = flattenMatrix(hm.matrix.matrix)
+            matrix_flatten = hm.matrix.matrix.flatten()
```

Comment:
the idea is to flatten the matrix only one. thus, be default is set to None. I replaced this by `if not matrix_flatten`